packages/scxa-marker-gene-heatmap/__test__/CalloutAlert.test.js

```diff
@@ -1,30 +1,35 @@
 import React from 'react'
-import Enzyme from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom' // for matcher like toBeInTheDocument
 import renderer from 'react-test-renderer'
-import { shallow } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
-
 import CalloutAlert from '../src/CalloutAlert'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-describe(`CalloutAlert`, () => {
+describe('CalloutAlert', () => {
```

Comment:
In other tests we are using backticks (`) and not single quotes (').
It would be better to use the same everywhere.
Thanks